packages/web-components/src/dialog/dialog.ts

```diff
@@ -94,10 +94,10 @@ export class Dialog extends FASTElement {
    * @returns boolean
    */
   public clickHandler(event: Event): boolean {
-    event.preventDefault();
```

Comment:
Do we want to write tests for this to make sure we don't break it and also preserve the weirdo AlertDialog behavior? Seems like an easy one to get tripped up on.

Reply:
Good call - I added a test that clicks a label for an input within a dialog, and checks if the input is properly focused.